website/src/pages/initiatives.js

```diff
@@ -71,37 +106,34 @@ const Community = () => {
           </div>
         </section>
         <section id="meetings">
-          <h3 className={styles.communitySubtitle}>Our Meetings</h3>
-          <div className={styles.communityRow}>
-            <div className={styles.communityMeeting}>
+        <h3 className={styles.communitySubtitle}>Our Meetings</h3>
+           <div>
+            <div> 
               <h5 className={styles.meetingTitle}>Past Meetings</h5>
```

Comment:
We no longer need the `Past Meetings` section. Only one calendar for the Meetings should be enough to track them.